vehicle-tracker/vehicle_tracker_app/lib/features/home/widgets/home_list_widget.dart

```diff
@@ -0,0 +1,76 @@
+import 'package:digit_components/digit_components.dart';
+import 'package:flutter/material.dart';
+import 'package:get/get.dart';
+
+import '../../../constants.dart';
+import '../../map/controllers/map_controllers.dart';
+
+class HomeListWidget extends StatelessWidget {
+  const HomeListWidget({super.key});
+
+  @override
+  Widget build(BuildContext context) {
+    final mapControllers = Get.find<MapControllers>();
+
+    return DigitCard(
+      child: Column(
+        crossAxisAlignment: CrossAxisAlignment.start,
+        children: [
+          Row(
+            children: [
+              const SizedBox(width: 8.0),
+              const Text("Fecal Sludge Management", style: TextStyle(fontSize: 18, fontWeight: FontWeight.bold)),
```

Comment:
Move to i 18 constants with translators and don't externally define the style use digit themes. i complete file

Reply:
alright